lib/kernel/test/interactive_shell_SUITE.erl

```diff
@@ -882,6 +889,93 @@ shell_invalid_ansi(_Config) ->
        "-eval","shell:prompt_func({interactive_shell_SUITE,prompt})."
       ]).
 
+shell_ignore_pager_commands(Config) ->
+    Name = peer:random_name(proplists:get_value(tc_path,Config)),
+
+    os:cmd("tmux new-window -n " ++ Name ++ " -d -- bash --norc"),
+
+    Peer = #{ name => Name,
+              post_process_args =>
+                        fun(["new-window","-n",_,"-d","--"|CmdAndArgs]) ->
+                                FlatCmdAndArgs =
+                                      lists:join(
+                                        " ",[[$',A,$'] || A <- CmdAndArgs]),
+                                ["send","-t",Name,lists:flatten(FlatCmdAndArgs),"Enter"]
+                        end
+            },
+    Term = start_tty([{peer, Peer}|Config]),
+    try
+        send_tty(Term, "h(file).\n"),
+        check_content(Term,"\\Qmore (y/n)? (y)\\E"),
+        send_tty(Term, "n\n"),
+        check_content(Term,"ok"),
+        send_tty(Term, "C-P"),
+        check_content(Term,"\\Qh(file).\\E"),
+        ok
+    after
+        stop_tty(Term),
+        ok
+    end.
+external_editor(Config) ->
+    Name = peer:random_name(proplists:get_value(tc_path,Config)),
+
+    os:cmd("tmux new-window -n " ++ Name ++ " -d -- bash --norc"),
+
+    Peer = #{ name => Name,
+              post_process_args =>
+                        fun(["new-window","-n",_,"-d","--"|CmdAndArgs]) ->
+                                FlatCmdAndArgs =
+                                      lists:join(
+                                        " ",[[$',A,$'] || A <- CmdAndArgs]),
+                                ["send","-t",Name,lists:flatten(FlatCmdAndArgs),"Enter"]
+                        end
+            },
+    Term = start_tty([{peer, Peer}|Config]),
+    try
+        send_tty(Term, "some text with\nnewline in it"),
+        check_content(Term,"2> some text with\n\\s+newline in it"),
+        send_tty(Term, "C-O"),
+        check_content(Term,"GNU nano \\d+[.]\\d+\\s+.+\\s+some text with\\s*\n\\s*newline in it"),
+        send_tty(Term, "still "),
+        send_tty(Term, "C-O"), %% save in nano
+        send_tty(Term, "Enter"),
+        send_tty(Term, "C-X"), %% quit in nano
+        check_content(Term,"still some text with\\s*\n\\s*newline in it"),
+        ok
+    after
+        stop_tty(Term),
+        ok
+    end.
```

Comment:
```suggestion
external_editor(Config) ->
    Term = start_tty(Config),
    try
        send_tty(Term,"os:putenv(\"EDITOR\",\"nano\").\n"),
        send_tty(Term, "some text with\nnewline in it"),
        check_content(Term,"2> some text with\n\\s+newline in it"),
        send_tty(Term, "C-O"),
        check_content(Term,"GNU nano [\\d.]+\\s+.+\\s+some text with\\s*\n\\s*newline in it"),
        send_tty(Term, "still "),
        send_tty(Term, "C-O"), %% save in nano
        send_tty(Term, "Enter"),
        send_tty(Term, "C-X"), %% quit in nano
        check_content(Term,"still some text with\\s*\n\\s*newline in it"),
        ok
    after
        stop_tty(Term),
        ok
    end.
```

No need to have the initial code and we need to set the EDITOR in the Erlang shell. Also the nano version can contain atleast 3 digits, maybe more so I made the regexp more lax.